src/test/java/com/github/paweladamski/httpclientmock/HttpClientMockBuilderTest.java

```diff
@@ -314,6 +321,97 @@ public void should_allow_different_host_then_default() throws IOException {
     assertThat(login, hasContent("login"));
     assertThat(google, hasContent("google"));
   }
+  
+  @Test
+  public void withFormParameter() throws IOException {
```

Comment:
Please try to make this more compact, I think that we don't need some many requests to check if `verifing` work.

Reply:
I removed some of the code (see 06bbc4b5592745b41bead8038787dc70d90d1629). Is that better?

I suppose most of it is redundant, since the bulk of the functionality is in the `UrlEncodedFormCondition` class.

Reply:
Yes, now this test looks much better.